src/main/resources/i18n/messages_es.properties

```diff
@@ -649,12 +649,12 @@ map.couldNotUnrank = No se pudo desrankear el mapa.
 # Fuzzy
 modVersion.ranked = Clasificatorio
 game.missionName = Misión
-settings.appearance.backgoundImage = Imagen de fondo
+settings.appearance.backgroundImage = Imagen de fondo
 settings.appearance.useDefaultImage = Usar imagen por defecto
 settings.appearance.chooseImage = Elige una imagen
 settings.general.theme.useNoImage = Sin imagen
 query.notContains = no contiene
 coop.unknownMission = Misión desconocida
 game.couldNotJoin = No se pudo unir a la partida con el ID ''{0}''.
-discord.disallowJoins = No permitir a jugadores unirse a través de Discord
+settings.general.disallowDiscordJoins = No permitir a jugadores unirse a través de Discord
```

Comment:
Will be overwritten rename in poeeditor

Reply:
Yes, unfortunately PoEditor won't detect the renames and the messages will have to be re-translated.
But it bothered me more that the naming convention wasn't followed ;-)

Reply:
Or, mabye we can rename the message key in PoEditor before importing the the message.properties?

Reply:
Whatever will sort itself